src/mlpack/core/data/load_arff_impl.hpp

```diff
@@ -30,6 +30,14 @@ void LoadARFF(const std::string& filename,
   std::ifstream ifs;
   ifs.open(filename, std::ios::in | std::ios::binary);
 
+  // if file is not open throw an error (file not found).
+  if (!ifs.is_open())
+  {
+    std::stringstream error;
+    error << filename << " Not Found.";
+    throw std::runtime_error(error.str());
```

Comment:
Do you think we can do away with the use of `std::stringstream` here? Also please change the error message to `File: abc.txt could not be opened` or something.

Reply:
@ShikharJ I didn't get how stringstream affects. And, I'll change the error message

Reply:
We simply don't require it in the first place, since only a single line needs to be printed. Plus readability is improved.

Reply:
@ShikharJ sure done.